### PR TITLE
feat: add class names to describe the audience

### PR DESCRIPTION
**Pull request by alexissmirnov**
- clearer, sharable URL "info" is better than "results" incase the URL is shared
- added the audience of the information

### CODE REVIEW

**Comment by alexissmirnov** [comment]
it would be nice to improve the layout of the audience and the change link

**Comment by alexissmirnov** [comment]
I don't understand the build break:

Creating an optimized production build...
The exported identifier "analytics" is not declared in Babel's scope tracker
as a JavaScript value binding, and "@babel/plugin-transform-typescript"
never encountered it as a TypeScript type declaration.
It will be treated as a JavaScript value.

File: src/i18n/translation.fr.js [hunk 1]
```diff
@@ -1,14 +1,33 @@
...
+    changeAudience: 'modifier',
+    audiencePrefix: "Les informations ci-dessous s'appliquent à "
   },
   welcomePage: {
     title: 'COVID-19: Ce que vous devez savoir',
     description:
       'Obtenez des informations fiables et personnalisées auprès de sources médicales canadiennes crédibles concernant le COVID-19.',
     button: 'Démarrer'
+  },
+  classes: {
```
Comment (vanessajones) [inline src/i18n/translation.fr.js]: Do these strings appear anywhere to the end user?

File: src/content/manifest.json [hunk 2]
```diff
@@ -1 +1,488 @@
-[]
+[
```
Comment (vanessajones) [inline src/content/manifest.json]: @alexissmirnov I think that this file is not suggested to be commited

Comment (vicrep) [reply src/content/manifest.json]: we should gitignore it then

File: src/pages/info.tsx [hunk 3]
```diff
@@ -45,53 +45,69 @@ const Title = styled.div`
...
   const query = useQuery()
   const { t } = useTranslation()
 
-  // parse /results?id=a,b,c -> [a, b, c]
+  // parse /info?id=a,b,c -> [a, b, c]
   const queryClasses = query.get('id')
   const classes = queryClasses ? queryClasses.split(',') : []
 
+  // build a comma separated, readable list of classes
+  var classString: string = ''
```
Comment (vicrep) [inline src/pages/info.tsx]:
```suggestion
  let classString: string = ''
```

var should _not_ be used when writing ES6

File: src/pages/info.tsx [hunk 4]
```diff
@@ -45,53 +45,69 @@ const Title = styled.div`
...
+  for (let c in classes) {
```
Comment (vicrep) [inline src/pages/info.tsx]: this can be expressed as such:
```
classString = classes.map(
  class => t(`classes.${class}`)
).join(', ')
```

Comment (vicrep) [reply src/pages/info.tsx]: then no need to do `substring` / `lastIndexOf`

**Pull request merged**